Backendless/Service/MessagingService.cs

```diff
@@ -72,13 +77,12 @@ public MessagingService()
 #endif
     }
 
-#if UNITY_ANDROID || UNITY_IPHONE || MOBILE
+#if (UNITY_ANDROID || UNITY_IPHONE || MOBILE) && NETSTANDARD2_1
```

Comment:
Why NETSTANDARD2_1? Plus it looks like .net standard will be going away: https://prnt.sc/yysixh
read more about it here: https://docs.microsoft.com/en-us/dotnet/standard/net-standard

Reply:
Fixed